src/inmanta/util.py

```diff
@@ -249,10 +252,12 @@ def api_boundary_json_encoder(o: object) -> Union[ReturnTypes, "JSONSerializable
         # Accross API boundaries, all naive datetime instances are assumed UTC. Returns ISO timestamp implicitly in UTC.
         return datetime_utc_isoformat(o, naive_utc=True)
 
-    return _custom_json_encoder(o)
+    return custom_json_encoder(o)
 
 
-def _custom_json_encoder(o: object) -> Union[ReturnTypes, "JSONSerializable"]:
+# deprecated as part of stable API: dropped in iso5
+@stable_api
```

Comment:
perhaps not make it @stable_api as it is going away

Reply:
Though we should indicate that this should be left as is in iso4, shouldn't we? My interpretation was that this was a deprecated part of the stable API, which should thus remain stable in iso4.
(I've created #2973 to more clearly indicate something like this in the future.)

Reply:
let's discuss on the meeting